docs/production/authentication/microsoft.mdx

```diff
@@ -181,6 +181,24 @@ You may need to restart your Mage server for the changes to take effect.
 
 </AccordionGroup>
 
+### Configurable environment variables
+To configure the environment for Microsoft Azure Cloud and Microsoft Azure Gov Cloud sign-in, 
+you need to set the following environment variables:
+1. You can specify the `AZURE_HOST` URLs based on your environment:
```

Comment:
@colefreeman This can be reworded to:
"You can set the `AZURE_HOST` environment variable to one of the following values based on your environment:"
```
Public cloud host url: "https://login.microsoftonline.com/"
China host url: "https://login.chinacloudapi.cn/"
Germany host url: "https://login.microsoftonline.de/"
Government host url: "https://login.microsoftonline.us/"
```

Reply:
After reading this a bit more closely, having it formatted like below can be confusing cuz users might think those are separate environment variables they need to set, when it's actually a single env var (`AZURE_HOST`) with different possible values.
```
AZURE_PUBLIC_CLOUD_HOST_URL="https://login.microsoftonline.com/"
AZURE_CHINA_HOST_URL="https://login.chinacloudapi.cn/"
AZURE_GERMANY_HOST_URL="https://login.microsoftonline.de/"
AZURE_GOVERNMENT_HOST_URL="https://login.microsoftonline.us/"
```

Reply:
@johnson-mage  Are you recommending to remove this?

Reply:
Yes, it should look like [this](https://github.com/mage-ai/mage-ai/pull/5352#discussion_r1727738429)

Reply:
I added the code block [here](https://github.com/mage-ai/mage-ai/pull/5352#discussion_r1727740322) as an example. When you write that code block out like that, `AZURE_PUBLIC_CLOUD_HOST_URL`, `AZURE_CHINA_HOST_URL`, `AZURE_GERMANY_HOST_URL`, and `AZURE_GOVERNMENT_HOST_URL` seem like separate environment variables, but they are just different values for the same environment variable (`AZURE_HOST`).

@colefreeman Does that make sense?

Reply:
Made this change to represent the method above and removed the trailing backslash (i.e. Public cloud host url: "https://login.microsoftonline.com")

Reply:
<img width="797" alt="Screenshot 2024-08-30 at 7 29 00 AM" src="https://github.com/user-attachments/assets/edc6a58f-d75a-46fb-8f83-520651106ff4">

Reply:
<img width="778" alt="Screenshot 2024-08-30 at 7 35 10 AM" src="https://github.com/user-attachments/assets/b4c34f34-482e-4cc2-a59d-8789171188b3">